### ISSUE TITLE
update fails due to `already exists`

### ISSUE THREAD

**Issue by yarikoptic** [dandi/dandisets#18]
```
2020-11-05T18:12:03-0500 [INFO    ] backups2datalad.py Syncing Dandiset 000003                                                                                                                                                                                                                                                                                                                                     
2020-11-05T18:12:04-0500 [INFO    ] dandi Traversing remote dandisets (000003) recursively                                                                                                                                                                                                                                                                                                                         
2020-11-05T18:12:04-0500 [INFO    ] backups2datalad.py Updating metadata file                                                                                                                                                                                                                                                                                                                                      
2020-11-05T18:12:05-0500 [WARNING ] dandi Multiple files for one item found; using oldest one                                                                                                                                                                                                                                                                                                                      
2020-11-05T18:12:05-0500 [WARNING ] dandi Multiple files for one item found; using oldest one                                                                                                                                                                                                                                                                                                                      
2020-11-05T18:12:05-0500 [WARNING ] dandi Multiple files for one item found; using oldest one                                                                                                                                                                                                                                                                                                                      
2020-11-05T18:12:08-0500 [WARNING ] dandi Ran into an empty item: {'_id': '5fa45767271e91aad754e522', '_modelType': 'item', 'baseParentId': '5e59bb0af19e820ab6ea6c62', 'baseParentType': 'collection', 'created': '2020-11-05T19:49:59.352000+00:00', 'creatorId': '5ee6c160155e470d51c94881', 'description': '', 'folderId': '5e7411e9368d3c79a8006c8c', 'meta': {}, 'name':                                     
'sub-YutaMouse42_ses-YutaMouse42-151112_behavior+ecephys.nwb', 'size': 0, 'updated': '2020-11-05T19:49:59.352000+00:00'}                                                                                                                                                                                                                                                                                           
2020-11-05T18:12:09-0500 [WARNING ] dandi Ran into an empty item: {'_id': '5fa4798e271e91aad754e54a', '_modelType': 'item', 'baseParentId': '5e59bb0af19e820ab6ea6c62', 'baseParentType': 'collection', 'created': '2020-11-05T22:15:42.342000+00:00', 'creatorId': '5ee6c160155e470d51c94881', 'description': '', 'folderId': '5fa47598271e91aad754e53f', 'meta': {}, 'name':                                     
'sub-YutaMouse45_ses-YutaMouse45-160305_behavior+ecephys.nwb', 'size': 0, 'updated': '2020-11-05T22:15:42.342000+00:00'}                                                                                                                                                                                                                                                                                           
2020-11-05T18:12:09-0500 [WARNING ] dandi Ran into an empty item: {'_id': '5fa47ab8271e91aad754e54d', '_modelType': 'item', 'baseParentId': '5e59bb0af19e820ab6ea6c62', 'baseParentType': 'collection', 'created': '2020-11-05T22:20:40.805000+00:00', 'creatorId': '5ee6c160155e470d51c94881', 'description': '', 'folderId': '5e741aba368d3c79a8006cbb', 'meta': {}, 'name':                                     
'sub-YutaMouse51_ses-YutaMouse51-160515_behavior+ecephys.nwb', 'size': 0, 'updated': '2020-11-05T22:20:40.805000+00:00'}                                                                                                                                                                                                                                                                                           
2020-11-05T18:12:09-0500 [WARNING ] dandi Ran into an empty item: {'_id': '5fa47f8d271e91aad754e551', '_modelType': 'item', 'baseParentId': '5e59bb0af19e820ab6ea6c62', 'baseParentType': 'collection', 'created': '2020-11-05T22:41:17.691000+00:00', 'creatorId': '5ee6c160155e470d51c94881', 'description': '', 'folderId': '5e741aba368d3c79a8006cbb', 'meta': {}, 'name':                                     
'sub-YutaMouse51_ses-YutaMouse51-160516_behavior+ecephys.nwb', 'size': 0, 'updated': '2020-11-05T22:41:17.691000+00:00'}                                                                                                                                                                                                                                                                                           
2020-11-05T18:12:09-0500 [WARNING ] dandi Ran into an empty item: {'_id': '5fa48199271e91aad754e555', '_modelType': 'item', 'baseParentId': '5e59bb0af19e820ab6ea6c62', 'baseParentType': 'collection', 'created': '2020-11-05T22:50:01.499000+00:00', 'creatorId': '5ee6c160155e470d51c94881', 'description': '', 'folderId': '5e741aba368d3c79a8006cbb', 'meta': {}, 'name':                                     
'sub-YutaMouse51_ses-YutaMouse51b160514_behavior+ecephys.nwb', 'size': 0, 'updated': '2020-11-05T22:50:01.499000+00:00'}                                                                                                                                                                                                                                                                                           
2020-11-05T18:12:09-0500 [WARNING ] dandi Ran into an empty item: {'_id': '5fa48209271e91aad754e559', '_modelType': 'item', 'baseParentId': '5e59bb0af19e820ab6ea6c62', 'baseParentType': 'collection', 'created': '2020-11-05T22:51:53.772000+00:00', 'creatorId': '5ee6c160155e470d51c94881', 'description': '', 'folderId': '5e741aba368d3c79a8006cbb', 'meta': {}, 'name':                                     
'sub-YutaMouse51_ses-YutaMouse51b160516_behavior+ecephys.nwb', 'size': 0, 'updated': '2020-11-05T22:51:53.772000+00:00'}                                                                                                                                                                                                                                                                                           
2020-11-05T18:12:09-0500 [WARNING ] dandi Ran into an empty item: {'_id': '5fa4825a271e91aad754e55b', '_modelType': 'item', 'baseParentId': '5e59bb0af19e820ab6ea6c62', 'baseParentType': 'collection', 'created': '2020-11-05T22:53:14.364000+00:00', 'creatorId': '5ee6c160155e470d51c94881', 'description': '', 'folderId': '5e741e51368d3c79a8006cc9', 'meta': {}, 'name':                                     
'sub-YutaMouse54_ses-YutaMouse54-160630_behavior+ecephys.nwb', 'size': 0, 'updated': '2020-11-05T22:53:14.364000+00:00'}                                                                                                                                                                                                                                                                                           
2020-11-05T18:12:09-0500 [WARNING ] dandi Ran into an empty item: {'_id': '5fa485a2271e91aad754e562', '_modelType': 'item', 'baseParentId': '5e59bb0af19e820ab6ea6c62', 'baseParentType': 'collection', 'created': '2020-11-05T23:07:14.831000+00:00', 'creatorId': '5ee6c160155e470d51c94881', 'description': '', 'folderId': '5e741e51368d3c79a8006cc9', 'meta': {}, 'name':                                     
'sub-YutaMouse54_ses-YutaMouse54-160701_behavior+ecephys.nwb', 'size': 0, 'updated': '2020-11-05T23:07:14.831000+00:00'}                                                                                                                                                                                                                                                                                           
2020-11-05T18:12:09-0500 [WARNING ] dandi Ran into an empty item: {'_id': '5fa485ef271e91aad754e564', '_modelType': 'item', 'baseParentId': '5e59bb0af19e820ab6ea6c62', 'baseParentType': 'collection', 'created': '2020-11-05T23:08:31.991000+00:00', 'creatorId': '5ee6c160155e470d51c94881', 'description': '', 'folderId': '5e741e51368d3c79a8006cc9', 'meta': {}, 'name':                                     
'sub-YutaMouse54_ses-YutaMouse54-160702_behavior+ecephys.nwb', 'size': 0, 'updated': '2020-11-05T23:08:31.991000+00:00'}                                                                                                                                                                                                                                                                                           
2020-11-05T18:12:09-0500 [WARNING ] dandi Ran into an empty item: {'_id': '5fa4843a271e91aad754e560', '_modelType': 'item', 'baseParentId': '5e59bb0af19e820ab6ea6c62', 'baseParentType': 'collection', 'created': '2020-11-05T23:01:14.914000+00:00', 'creatorId': '5ee6c160155e470d51c94881', 'description': '', 'folderId': '5e741e51368d3c79a8006cc9', 'meta': {}, 'name':                                     
'sub-YutaMouse54_ses-YutaMouse54b160701_behavior+ecephys.nwb', 'size': 0, 'updated': '2020-11-05T23:01:14.914000+00:00'}                                                                                                                                                                                                                                                                                           
2020-11-05T18:12:10-0500 [INFO    ] backups2datalad.py Syncing asset /sub-YutaMouse20/sub-YutaMouse20_ses-YutaMouse20-140321_behavior+ecephys.nwb                                                                                                                                                                                                                                                                  
2020-11-05T18:12:11-0500 [INFO    ] backups2datalad.py Asset in dataset, and hash shows modification; will update                                                                                                                                                                                                                                                                                                  
2020-11-05T18:12:11-0500 [INFO    ] backups2datalad.py Asset not found in assetstore; downloading from https://dandiarchive.s3.amazonaws.com/girder-assetstore/99/31/99311ec02ab2402f87b8ceda2cb8348d?versionId=6Ws69yYxdCA9oj5TNPb_oHs2xSaMNnSm                                                                                                                                                                   
[INFO] Downloading 'https://dandiarchive.s3.amazonaws.com/girder-assetstore/99/31/99311ec02ab2402f87b8ceda2cb8348d?versionId=6Ws69yYxdCA9oj5TNPb_oHs2xSaMNnSm' into '/mnt/backup/dandi/dandisets/000003/sub-YutaMouse20/sub-YutaMouse20_ses-YutaMouse20-140321_behavior+ecephys.nwb'                                                                                                                               
2020-11-05T18:12:11-0500 [INFO    ] datalad.downloaders Downloading 'https://dandiarchive.s3.amazonaws.com/girder-assetstore/99/31/99311ec02ab2402f87b8ceda2cb8348d?versionId=6Ws69yYxdCA9oj5TNPb_oHs2xSaMNnSm' into '/mnt/backup/dandi/dandisets/000003/sub-YutaMouse20/sub-YutaMouse20_ses-YutaMouse20-140321_behavior+ecephys.nwb'                                                                              
2020-11-05T18:12:11-0500 [ERROR   ] backups2datalad.py Operation failed with exception:                                                                                                                                                                                                                                                                                                                            
Traceback (most recent call last):                                                                                                                                                                                                                                                                                                                                                                                 
  File "tools/backups2datalad.py", line 462, in dandi_logging                                                                                                                                                                                                                                                                                                                                                      
    yield logfile                                                                                                                                                                                                                                                                                                                                                                                                  
  File "tools/backups2datalad.py", line 321, in sync_dataset                                                                                                                                                                                                                                                                                                                                                       
    ds.download_url(urls=bucket_url, path=deststr)                                                                                                                                                                                                                                                                                                                                                                 
  File "/home/dandi/miniconda3/envs/dandi-cli-dev/lib/python3.8/site-packages/datalad/distribution/dataset.py", line 503, in apply_func                                                                                                                                                                                                                                                                            
    return f(**kwargs)                                                                                                                                                                                                                                                                                                                                                                                             
  File "/home/dandi/miniconda3/envs/dandi-cli-dev/lib/python3.8/site-packages/datalad/interface/utils.py", line 494, in eval_func                                                                                                                                                                                                                                                                                  
    return return_func(generator_func)(*args, **kwargs)                                                                                                                                                                                                                                                                                                                                                            
  File "/home/dandi/miniconda3/envs/dandi-cli-dev/lib/python3.8/site-packages/datalad/interface/utils.py", line 482, in return_func                                                                                                                                                                                                                                                                                
    results = list(results)                                                                                                                                                                                                                                                                                                                                                                                        
  File "/home/dandi/miniconda3/envs/dandi-cli-dev/lib/python3.8/site-packages/datalad/interface/utils.py", line 467, in generator_func                                                                                                                                                                                                                                                                             
    raise IncompleteResultsError(                                                                                                                                                                                                                                                                                                                                                                                  
datalad.support.exceptions.IncompleteResultsError: Command did not complete successfully. 1 failed:                                                                                                                                                                                                                                                                                                                
[{'action': 'download_url',                                                                                                                                                                                                                                                                                                                                                                                        
  'message': 'File '                                                                                                                                                                                                                                                                                                                                                                                               
             '/mnt/backup/dandi/dandisets/000003/sub-YutaMouse20/sub-YutaMouse20_ses-YutaMouse20-140321_behavior+ecephys.nwb '                                                                                                                                                                                                                                                                                     
             'already exists [base.py:_download:445]',                                                                                                                                                                                                                                                                                                                                                             
  'path': '/mnt/backup/dandi/dandisets/000003/sub-YutaMouse20/sub-YutaMouse20_ses-YutaMouse20-140321_behavior+ecephys.nwb',                                                                                                                                                                                                                                                                                        
  'status': 'error',                                                                                                                                                                                                                                                                                                                                                                                               
  'type': 'file'}]                                                                                                                                                                                                                                                                                                                                                                                                 
Traceback (most recent call last):                                                                                                                                                                                                                                                                                                                                                                                 
  File "tools/backups2datalad.py", line 471, in <module>                                                                                                                                                                                                                                                                                                                                                           
    main()                                                                                                                                                                                                                                                                                                                                                                                                         
  File "/home/dandi/miniconda3/envs/dandi-cli-dev/lib/python3.8/site-packages/click/core.py", line 829, in __call__                                                                                                                                                                                                                                                                                                
    return self.main(*args, **kwargs)                                                                                                                                                                                                                                                                                                                                                                              
  File "/home/dandi/miniconda3/envs/dandi-cli-dev/lib/python3.8/site-packages/click/core.py", line 782, in main                                                                                                                                                                                                                                                                                                    
    rv = self.invoke(ctx)                                                                                                                                                                                                                                                                                                                                                                                          
  File "/home/dandi/miniconda3/envs/dandi-cli-dev/lib/python3.8/site-packages/click/core.py", line 1066, in invoke                                                                                                                                                                                                                                                                                                 
    return ctx.invoke(self.callback, **ctx.params)                                                                                                                                                                                                                                                                                                                                                                 
  File "/home/dandi/miniconda3/envs/dandi-cli-dev/lib/python3.8/site-packages/click/core.py", line 610, in invoke                                                                                                                                                                                                                                                                                                  
    return callback(*args, **kwargs)                                                                                                                                                                                                                                                                                                                                                                               
  File "tools/backups2datalad.py", line 96, in main                                                                                                                                                                                                                                                                                                                                                                
    DatasetInstantiator(                                                                                                                                                                                                                                                                                                                                                                                           
  File "tools/backups2datalad.py", line 170, in run                                                                                                                                                                                                                                                                                                                                                                
    if self.sync_dataset(did, ds):                                                                                                                                                                                                                                                                                                                                                                                 
  File "tools/backups2datalad.py", line 321, in sync_dataset                                                                                                                                                                                                                                                                                                                                                       
    ds.download_url(urls=bucket_url, path=deststr)                                                                                                                                                                                                                                                                                                                                                                 
  File "/home/dandi/miniconda3/envs/dandi-cli-dev/lib/python3.8/site-packages/datalad/distribution/dataset.py", line 503, in apply_func                                                                                                                                                                                                                                                                            
    return f(**kwargs)                                                                                                                                                                                                                                                                                                                                                                                             
  File "/home/dandi/miniconda3/envs/dandi-cli-dev/lib/python3.8/site-packages/datalad/interface/utils.py", line 494, in eval_func                                                                                                                                                                                                                                                                                  
    return return_func(generator_func)(*args, **kwargs)                                                                                                                                                                                                                                                                                                                                                            
  File "/home/dandi/miniconda3/envs/dandi-cli-dev/lib/python3.8/site-packages/datalad/interface/utils.py", line 482, in return_func                                                                                                                                                                                                                                                                                
    results = list(results)                                                                                                                                                                                                                                                                                                                                                                                        
  File "/home/dandi/miniconda3/envs/dandi-cli-dev/lib/python3.8/site-packages/datalad/interface/utils.py", line 467, in generator_func                                                                                                                                                                                                                                                                             
    raise IncompleteResultsError(                                                                                                                                                                                                                                                                                                                                                                                  
datalad.support.exceptions.IncompleteResultsError: Command did not complete successfully. 1 failed:                                                                                                                                                                                                                                                                                                                
[{'action': 'download_url',                                                                                                                                                                                                                                                                                                                                                                                        
  'message': 'File '                                                                                                                                                                                                                                                                                                                                                                                               
             '/mnt/backup/dandi/dandisets/000003/sub-YutaMouse20/sub-YutaMouse20_ses-YutaMouse20-140321_behavior+ecephys.nwb '                                                                                                                                                                                                                                                                                     
             'already exists [base.py:_download:445]',                                                                                                                                                                                                                                                                                                                                                             
  'path': '/mnt/backup/dandi/dandisets/000003/sub-YutaMouse20/sub-YutaMouse20_ses-YutaMouse20-140321_behavior+ecephys.nwb',                                                                                                                                                                                                                                                                                        
  'status': 'error',                                                                                                                                                                                                                                                                                                                                                                                               
  'type': 'file'}]                                                                                                                                                                                                                                                                                                                                                                                                 
                      
```

altogether -- I think we need to at least make backup (of assetstore) as frequent as the runs of this script.  Actually better to bundle them together and possibly just skip dandisets for which there is an item with file not yet in assetstore (they all should be).

yet to analyze this specific case . will turn off the cron job for the script for now

**Comment by yarikoptic** [comment]
<details>
<summary>This issue persists, although logs are a bit different with more up to date dandi-cli etc</summary> 

```shell
(dandisets) dandi@drogon:/mnt/backup/dandi/dandisets$ git show
commit 47c7506eef3f8e28c5bc9cebf7cb0a48ca9365ed (HEAD -> master, github/master)
Merge: b5e2192 cf80d5c
Author: DANDI Meta-user <dandi@mit.edu>
Date:   Mon Nov 9 17:14:43 2020 -0500

    Merge branch 'master' of github.com:dandi/dandisets

(dandisets) dandi@drogon:/mnt/backup/dandi/dandisets$ tools/backups2datalad.py --backup-remote dandi-dandisets-dropbox --gh-org dandisets -J 5 /mnt/backup/dandi/dandiarchive-s3-backup/ /mnt/backup/dandi/dandisets 000003
2020-11-09T17:21:16-0500 [INFO    ] backups2datalad.py Syncing Dandiset 000003
2020-11-09T17:21:17-0500 [INFO    ] dandi Traversing remote dandisets (000003) recursively
2020-11-09T17:21:17-0500 [INFO    ] backups2datalad.py Updating metadata file
2020-11-09T17:21:18-0500 [WARNING ] dandi Multiple files for one item found; using oldest one
2020-11-09T17:21:18-0500 [WARNING ] dandi Multiple files for one item found; using oldest one
2020-11-09T17:21:18-0500 [WARNING ] dandi Multiple files for one item found; using oldest one
2020-11-09T17:21:22-0500 [INFO    ] backups2datalad.py Syncing asset /sub-YutaMouse20/sub-YutaMouse20_ses-YutaMouse20-140321_behavior+ecephys.nwb
2020-11-09T17:21:23-0500 [INFO    ] backups2datalad.py Asset in dataset, and hash shows modification; will update
2020-11-09T17:21:23-0500 [INFO    ] backups2datalad.py Asset not found in assetstore; downloading from https://dandiarchive.s3.amazonaws.com/girder-assetstore/99/31/99311ec02ab2402f87b8ceda2cb8348d?versionId=6Ws69yYxdCA9oj5TNPb_oHs2xSaMNnSm
[INFO   ] Downloading 'https://dandiarchive.s3.amazonaws.com/girder-assetstore/99/31/99311ec02ab2402f87b8ceda2cb8348d?versionId=6Ws69yYxdCA9oj5TNPb_oHs2xSaMNnSm' into '/mnt/backup/dandi/dandisets/000003/sub-YutaMouse20/sub-YutaMouse20_ses-YutaMouse20-140321_behavior+ecephys.nwb' 
2020-11-09T17:21:23-0500 [INFO    ] datalad.downloaders Downloading 'https://dandiarchive.s3.amazonaws.com/girder-assetstore/99/31/99311ec02ab2402f87b8ceda2cb8348d?versionId=6Ws69yYxdCA9oj5TNPb_oHs2xSaMNnSm' into '/mnt/backup/dandi/dandisets/000003/sub-YutaMouse20/sub-YutaMouse20_ses-YutaMouse20-140321_behavior+ecephys.nwb'
2020-11-09T17:21:23-0500 [ERROR   ] backups2datalad.py Operation failed with exception:
Traceback (most recent call last):
  File "tools/backups2datalad.py", line 466, in dandi_logging
    yield logfile
  File "tools/backups2datalad.py", line 325, in sync_dataset
    ds.download_url(urls=bucket_url, path=deststr)
  File "/home/dandi/miniconda3/envs/dandisets/lib/python3.8/site-packages/datalad/distribution/dataset.py", line 503, in apply_func
    return f(**kwargs)
  File "/home/dandi/miniconda3/envs/dandisets/lib/python3.8/site-packages/datalad/interface/utils.py", line 495, in eval_func
    return return_func(generator_func)(*args, **kwargs)
  File "/home/dandi/miniconda3/envs/dandisets/lib/python3.8/site-packages/datalad/interface/utils.py", line 483, in return_func
    results = list(results)
  File "/home/dandi/miniconda3/envs/dandisets/lib/python3.8/site-packages/datalad/interface/utils.py", line 468, in generator_func
    raise IncompleteResultsError(
datalad.support.exceptions.IncompleteResultsError: Command did not complete successfully. 1 failed:
[{'action': 'download_url',
  'message': 'File '
             '/mnt/backup/dandi/dandisets/000003/sub-YutaMouse20/sub-YutaMouse20_ses-YutaMouse20-140321_behavior+ecephys.nwb '
             'already exists [base.py:_download:445]',
  'path': '/mnt/backup/dandi/dandisets/000003/sub-YutaMouse20/sub-YutaMouse20_ses-YutaMouse20-140321_behavior+ecephys.nwb',
  'status': 'error',
  'type': 'file'}]
Traceback (most recent call last):
  File "tools/backups2datalad.py", line 475, in <module>
    main()
  File "/home/dandi/miniconda3/envs/dandisets/lib/python3.8/site-packages/click/core.py", line 829, in __call__
    return self.main(*args, **kwargs)
  File "/home/dandi/miniconda3/envs/dandisets/lib/python3.8/site-packages/click/core.py", line 782, in main
    rv = self.invoke(ctx)
  File "/home/dandi/miniconda3/envs/dandisets/lib/python3.8/site-packages/click/core.py", line 1066, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/dandi/miniconda3/envs/dandisets/lib/python3.8/site-packages/click/core.py", line 610, in invoke
    return callback(*args, **kwargs)
  File "tools/backups2datalad.py", line 96, in main
    DatasetInstantiator(
  File "tools/backups2datalad.py", line 170, in run
    if self.sync_dataset(did, ds):
  File "tools/backups2datalad.py", line 325, in sync_dataset
    ds.download_url(urls=bucket_url, path=deststr)
  File "/home/dandi/miniconda3/envs/dandisets/lib/python3.8/site-packages/datalad/distribution/dataset.py", line 503, in apply_func
    return f(**kwargs)
  File "/home/dandi/miniconda3/envs/dandisets/lib/python3.8/site-packages/datalad/interface/utils.py", line 495, in eval_func
    return return_func(generator_func)(*args, **kwargs)
  File "/home/dandi/miniconda3/envs/dandisets/lib/python3.8/site-packages/datalad/interface/utils.py", line 483, in return_func
    results = list(results)
  File "/home/dandi/miniconda3/envs/dandisets/lib/python3.8/site-packages/datalad/interface/utils.py", line 468, in generator_func
    raise IncompleteResultsError(
datalad.support.exceptions.IncompleteResultsError: Command did not complete successfully. 1 failed:
[{'action': 'download_url',
  'message': 'File '
             '/mnt/backup/dandi/dandisets/000003/sub-YutaMouse20/sub-YutaMouse20_ses-YutaMouse20-140321_behavior+ecephys.nwb '
             'already exists [base.py:_download:445]',
  'path': '/mnt/backup/dandi/dandisets/000003/sub-YutaMouse20/sub-YutaMouse20_ses-YutaMouse20-140321_behavior+ecephys.nwb',
  'status': 'error',
  'type': 'file'}]

```
</details>

-  we need more informative "dandi Multiple files for one item found; using oldest one" -- for which item? I have checked https://girder.dandiarchive.org/#item/5fa42f0b271e91aad754e4b5 for the one which fails the run, but there is only one run so we still don't know which asset in girder to check to be "cleaned up". Filed https://github.com/dandi/dandi-cli/issues/270
- `Asset not found in assetstore; downloading from ` is worrisome -- suggests that the backup `datalad crawl` run managed to miss some files.... I don't know yet how/why -- I will investigate (getting a full listing of s3 ATM to see what else we had missed :-( )
- I guess backup script needs fixup to have first target file removed before downloading a fresh copy? (passing this issue over to @jwodder)

**Comment by yarikoptic** [comment]
Fixed by #25
